doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -0,0 +1,1352 @@
+   - lists all the VLANs learnt from the specified remote ip or all the remotes.  (APP DB view) 
+   - VLAN, RemoteVTEP, VNI are the columns
+
+   show vxlan remote_vni all
+   +---------+--------------+-------+
+   | VLAN    | RemoteVTEP   |   VNI |
+   +=========+==============+=======+
+   | Vlan101 | 3.3.3.3      |  1001 |
+   +---------+--------------+-------+
+   | Vlan101 | 4.4.4.4      |  1001 |
+   +---------+--------------+-------+
+   Total count : 2
+   
+   show vxlan remote_vni 3.3.3.3
+   +---------+--------------+-------+
+   | VLAN    | RemoteVTEP   |   VNI |
+   +=========+==============+=======+
+   | Vlan101 | 3.3.3.3      |  1001 |
+   +---------+--------------+-------+
+   Total count : 1
+
+
+```
+
+### 5.2 KLISH CLI
+
+#### 5.2.1 Configuration Commands
+
+```
+1. VTEP Source IP configuration
+   - switch(config) interface vxlan <vtepname>
+   - switch(config-if-vtep1) [no] source-ip  <src_ipv4>
+   - <src_ipv4> is an IPV4 address in dotted notation A.B.C.D
+2. VLAN VNI Mapping configuration
+   - switch(config-if-vtep1) [no] map vlan <vidstart> vni <vnistart>  count <n>
+   - <n> is the number of mappings being configured. 
+   - <vidstart>, <vnistart> are the starting VID and VNID. 
+   - count is optional and when specified maps contigous sets of VIDs to contigous VNIDs
+4. VRF VNI Mapping configuration
+   - switch(config-if-vtep1) [no] map vrf VRF-Blue vni 10001
+5. Neighbor suppression
+   - switch(config) interface Vlan <vlan-id-xxx>
+   - switch(conf-if-vlanxxx)# [no] neigh-suppress
+   - This command will suppress both ARP & ND flooding over the tunnel when Vlan is extended.
+
+```
+
+#### 5.2.2 Show Commands
+
+     The show command outputs shall be from the exec mode of the KLISH shell and has the same
+     content as the show commands in the Click show section. 
+     In addition the following commands will be supported.
+
+```
+1. show vlan id and brief will be enhanced to display DIP. (tunnel members from  APP DB)
+
+2. show mac /show mac -v will be enhanced for EVPN MACs.
+   - Port column will display DIP. (ASIC DB view)
+   - Type column will display EVPN_static/EVPN_dynamic
+
+3. show NeighbourSuppressStatus to display Neighbor suppression status.
+
+```
+
+#### 5.2.3 Validations 
+
+```
+- VLAN to be configured before map vlan vni command is executed. 
+- VLAN cannot be deleted if there is a vlan vni map. 
+- source to be configured before  map vlan vni command is executed.
+- source cannot be removed if there are mappings already present. 
+- interface vxlan cannot be deleted if evpn nvo is configured. 
+- VRF is already configured    
+- VNI VLAN map should be configured prior to VRF VNI map configuration, since VNI should be both L2 and L3.
+- VNI VLAN map cannot be deleted if there is corresponding VRF VNI map.
+
+```
+
+## 6 Serviceability and Debug
+
+The existing logging mechanisms shall be used. The show commands and the redis table
+dumps can be used as debug aids.
+
+## 7 Warm Reboot Support
```

Comment:
This is an enhancement to existing warmboot sequence and may need to be taken with warmboot forum. I'll check and get back on any feedback.